code/missionui/redalert.cpp

```diff
@@ -1034,7 +1076,9 @@ void red_alert_bash_wingman_status()
 			wingp->current_wave++;
 			wingp->red_alert_skipped_ships += wingp->wave_count;
 
-            if (wingp->num_waves == 0)
+			bool waves_spent = wingp->current_wave >= wingp->num_waves;
```

Comment:
I think this should be >, not >=.  The `current_wave` field is 1-based.
```suggestion
			bool waves_spent = wingp->current_wave > wingp->num_waves;
```